public/catalog/v1.0.0/openapi.json

```diff
@@ -1454,6 +1454,13 @@
             "type": "string",
             "title": "External url",
             "description": "The external url of the service instance used with relation to this order item"
+          },
+          "insights_request_id":{
+            "type": "string",
+            "title": "Insights Request ID ",
+            "example": "insights-request-id-string",
```

Comment:
@lindgrenj6 A good example would be 364498f142194beba576833d7303abe5

e.g.
x-rh-insights-request-id: 364498f142194beba576833d7303abe5